icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -640,6 +640,29 @@ class U_I18N_API Precision : public UMemory {
      */
     static IncrementPrecision increment(double roundingIncrement);
 
+#ifndef U_HIDE_DRAFT_API
```

Comment:
The API was approved to be:

```
#ifndef U_HIDE_DRAFT_API
    /**
     * Version of `Precision::increment()` that takes an integer at a particular power of 10.
     *
     * To round to the nearest 0.5 and display 2 fraction digits, with this function, you should write one of the following:
     *
     * <pre>
     * Precision::incrementExact(5, -1).withMinFraction(2)
     * Precision::incrementExact(50, -2).withMinFraction(2)
     * Precision::incrementExact(50, -2)
     * </pre>
     *
     * Same as in ICU4J Precision.increment(new BigDecimal(“0.50”)) ...
     *
     * This behavior is modeled after ECMA-402. For more information, see:
     * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#roundingincrement
     *
     * @param mantissa
     *            The increment to which to round numbers.
     * @param magnitude
     *            The power of 10 of the ones digit of roundingIncrement.
     * @return A precision for chaining or passing to the NumberFormatter precision() setter.
     * @draft ICU 71
     */
    static IncrementPrecision incrementExact(uint64_t mantissa, int16_t magnitude);
#endif // U_HIDE_DRAFT_API
```

I need to update the PR with the new signature.

Note that the actual behavior is not changed, so the PR is still reviewable.